services/web/project/repository/token_repository.py

```diff
@@ -0,0 +1,27 @@
+from flask import current_app
+from ..model import Token
+
+
+def list_all():
+    return Token.query.all()
+
+def get_by_id(id):
+    return Token.query.get(id)
+
+def save(token):
+    current_app.db.session.add(token)
+    current_app.db.session.commit()
+    return token
+
+def update(id, token):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'id'](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5161591)